hypervisor/src/mshv/mod.rs

```diff
@@ -370,11 +370,21 @@ impl hypervisor::Hypervisor for MshvHypervisor {
         self.create_vm_with_type(vm_type)
     }
     #[cfg(target_arch = "x86_64")]
+
     ///
     /// Get the supported CpuID
     ///
     fn get_supported_cpuid(&self) -> hypervisor::Result<Vec<CpuIdEntry>> {
-        Ok(Vec::new())
+        let mut cpuid: Vec<CpuIdEntry> = Vec::new();
+        let functions: [u32; 2] = [0x1, 0xb];
```

Comment:
You can do `[u32; _]` and let the compiler figure out the length. The original code is fine, too.

Reply:
Interesting. Using `_` for array lengths is not stable. Forget about my original comment.